starknet_py/net/schemas/rpc.py

```diff
@@ -1,3 +1,5 @@
+# pylint: disable=too-many-lines
```

Comment:
we could think about splitting this file into the smallest. but in separate task

Reply:
https://github.com/software-mansion/starknet.py/issues/1316